File: dataset/output-bk/MISTRAL_codestral-latest/Task128_MISTRAL_codestral-latest.java
import java.util.Random;

public class RandomTokenGenerator {
    private static final String CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
    private static final int TOKEN_LENGTH = 10;

    public static void main(String[] args) {
        System.out.println(generateRandomToken());
    }

    private static String generateRandomToken() {
        Random random = new Random();
        StringBuilder token = new StringBuilder(TOKEN_LENGTH);

        for (int i = 0; i < TOKEN_LENGTH; i++) {
            int randomIndex = random.nextInt(CHARACTERS.length());
            token.append(CHARACTERS.charAt(randomIndex));
        }

        return token.toString();
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task128_MISTRAL_codestral-latest.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

const char CHARACTERS[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
const int TOKEN_LENGTH = 10;

void generateRandomToken(char* token) {
    srand(time(0));

    for (int i = 0; i < TOKEN_LENGTH; i++) {
        int randomIndex = rand() % (sizeof(CHARACTERS) - 1);
        token[i] = CHARACTERS[randomIndex];
    }

    token[TOKEN_LENGTH] = '\0';
}

int main() {
    char token[TOKEN_LENGTH + 1];
    generateRandomToken(token);
    printf("%s
", token);
    return 0;
}